joomla/joomla-cms
# [4.4] tags learn more link

Pull Request for Issue #42812 .

### Summary of Changes
Changes the link to the docs to one for j4 and not j3 which is more detailed


### Testing Instructions
With no tags you will see the emtpy state view and the learn more button

![image](https://github.com/joomla/joomla-cms/assets/1296369/7d8f6619-7dae-4304-bd15-5e401c856d8b)


### Actual result BEFORE applying this Pull Request
j3 page opens


### Expected result AFTER applying this Pull Request
j4 page opens


### Link to documentations
Please select:
